:art: (docs & ci) rework scripts and templates (#40)
adfinis-sygroup/helm-charts: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---
**Affected chart**

> Tell us which chart this bug is about.

Reviewer: ```suggestion
Tell us which chart is affected by the bug.
```

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Logs**

> If applicable, add logs or screenshots to help explain your problem.

Reviewer: ```suggestion
If applicable, add logs or screenshots to help illustrate the problem.
```

**Affected version**
Which version of the chart were you using.

**Values**

> Let us know of any values you are setting.

Reviewer: ```suggestion
Let us know of any specific values you have configured.
```

**Environment**

> What version of Kubernetes from what vendor are you using.

Reviewer 2: shouldn't we also suggest adding the helm version? Perhaps even adding example code to get the output we need. 

i.e. `kubectl version` and `helm version`
